src/imageOperations/multiReceiptsExtractor.ts

```diff
@@ -0,0 +1,73 @@
+import { PDFDocument, PDFImage, PDFPage } from "pdf-lib";
+import { MindeeError, MindeeMimeTypeError } from "../errors";
+import { Polygon, getMinMaxX, getMinMaxY } from "../geometry";
+import { MultiReceiptsDetectorV1 } from "../product";
+import { ExtractedImage } from "./extractedImage";
+import { ExtractedMultiReceiptImage } from "./extractedMultiReceiptImage";
+import { LocalInputSource } from "../input/base";
+
+async function addPage(
+  pdfPage: PDFPage,
+  boundingBox: Polygon,
+  pageId: number,
+  receiptId: number) {
+  const { width, height } = pdfPage.getSize();
+  const receiptPdf = await PDFDocument.create();
+
+  const newWidth = width * (getMinMaxX(boundingBox).max - getMinMaxX(boundingBox).min);
+  const newHeight = height * (getMinMaxY(boundingBox).max - getMinMaxY(boundingBox).min);
+  //Note: PDF-lib seems to invert y coordinates, giving us the following horror:
+  const croppedReceipt = await receiptPdf.embedPage(pdfPage, {
+    left: getMinMaxX(boundingBox).min * width,
+    right: getMinMaxX(boundingBox).max * width,
+    top: height - (getMinMaxY(boundingBox).min * height),
+    bottom: height - (getMinMaxY(boundingBox).max * height),
+
+  });
+  const receiptPage = receiptPdf.addPage([newWidth, newHeight]);
+  receiptPage.drawPage(croppedReceipt,
+    {
+      width: newWidth,
+      height: newHeight,
+    });
+  const receiptBytes = await receiptPdf.save();
+  return new ExtractedMultiReceiptImage(receiptBytes, pageId, receiptId);
+}
+
+async function loadPdfDoc(inputFile: LocalInputSource) {
+  let pdfDoc: PDFDocument;
+  if (["image/jpeg", "image/jpg", "image/png"].includes(inputFile.mimeType)) {
```

Comment:
invert the check, if not one of the good types throw an error directly rather than hiding it all the way at the end of the function

Reply:
if not includes throw